### PR TITLE
Update CODEOWNERS with team

### PR DESCRIPTION
**Pull request by mehulkar**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
-*                 @jaredpalmer @gsoltis @gaspar09 @nathanhammond @tknickman @chris-olszewski @mehulkar
+* @turbo-oss
```
Comment (mehulkar) [inline .github/CODEOWNERS:4]: 🤔  https://github.com/orgs/vercel/teams/turbo-oss or is it case sensitivey

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
-*                 @jaredpalmer @gsoltis @gaspar09 @nathanhammond @tknickman @chris-olszewski @mehulkar
+* @Turbo-OSS
```
Comment (mehulkar) [inline .github/CODEOWNERS:4]: not sure why GH is flagging this. @jaredpalmer does this team have Write access to the repo?

Comment (tknickman) [reply .github/CODEOWNERS:4]: You need to scope this and lowercase it. 

```suggestion
* @vercel/turbo-oss 
```

**Review by tknickman — CHANGES_REQUESTED** [review]

**Comment by tknickman** [comment]
I'm just going to approve this, and since it _shouldn't_ auto merge anymore (thanks!) you can feel free to merge when it's working!

**Review by tknickman — APPROVED** [review]

**Pull request merged**